hypha/apply/dashboard/templates/dashboard/dashboard.html

```diff
@@ -14,10 +14,12 @@
         {% block page_header %}
             <h1 class="gamma heading heading--no-margin heading--bold">{% trans "Dashboard" %}</h1>
         {% endblock %}
-        <a href="{% url 'wagtailadmin_home' %}" class="button button--primary button--arrow-pixels-white">
-            {% trans "Apply admin" %}
-            <svg><use xlink:href="#arrow-head-pixels--solid"></use></svg>
-        </a>
+        {% if has_wagtail_admin_access %}
```

Comment:
Can we use `{% if perms.wagtailadmin.access_admin %}` directly? Any downsides to that?

Reply:
Fixed up!  I think that's clear enough what it means (the direct-access name is *very slightly* less clear about what we're checking IMO, but not enough to warrant the extra field)